ui/pages/settings/networks-tab/networks-tab.constants.js

```diff
@@ -45,13 +44,16 @@ const defaultNetworksData = [
     blockExplorerUrl: 'https://sepolia.etherscan.io',
   },
   {
-    labelKey: NETWORK_TYPES.LINEA_TESTNET,
+    labelKey: NETWORK_TYPES.LINEA_GOERLI,
     iconColor: '#234FD5',
-    providerType: NETWORK_TYPES.LINEA_TESTNET,
-    rpcUrl: LINEA_TESTNET_RPC_URL,
-    chainId: CHAIN_IDS.LINEA_TESTNET,
-    ticker: TEST_NETWORK_TICKER_MAP[NETWORK_TYPES.LINEA_TESTNET],
-    blockExplorerUrl: 'https://explorer.goerli.linea.build',
+    providerType: NETWORK_TYPES.LINEA_GOERLI,
+    rpcUrl: getRpcUrl({
+      network: NETWORK_TYPES.LINEA_GOERLI,
+      excludeProjectId: true,
+    }),
+    chainId: CHAIN_IDS.LINEA_GOERLI,
+    ticker: TEST_NETWORK_TICKER_MAP[NETWORK_TYPES.LINEA_GOERLI],
+    blockExplorerUrl: 'https://goerli.lineascan.build',
```

Comment:
Likewise for this URL - we can wait until it's live

Reply:
This block explorer is working :)